code-of-conduct.md

```diff
@@ -14,10 +14,10 @@ We value diverse opinions, but we value respectful behavior more.
 
 Respectful behavior includes:
 
-* Being considerate, kind, constructive, and helpful.
-* Not engaging in demeaning, discriminatory, harassing, hateful, sexualized, or
+*   Being considerate, kind, constructive, and helpful.
```

Comment:
These are standard files that are included verbatim.
I think they should be excluded from the linter review.

Reply:
Updated the linter action to ignore the Code of Conduct file.